ui/src/app/views/home-page/component.ts

```diff
@@ -43,26 +45,33 @@ export class HomePageComponent implements OnInit {
   private workspaceResearchPurposeFilter = new WorkspaceResearchPurposeFilter();
   private workspaceNameComparator = new WorkspaceNameComparator();
 
+  errorText: string;
   workspaceList: WorkspaceResponse[] = [];
   workspacesLoading = false;
   workspaceAccessLevel = WorkspaceAccessLevel;
   constructor(
+      private errorHandlingService: ErrorHandlingService,
       private route: ActivatedRoute,
       private router: Router,
       private workspacesService: WorkspacesService,
   ) {}
   ngOnInit(): void {
+    this.errorText = null;
```

Comment:
No need to do this, unless you're checking somewhere specifically for `null` as opposed to `undefined`.  Otherwise this doesn't alter the boolean context, `this.errorText` still evaluates to `false`.

If you _are_ checking against `null` specifically (and that can't be done away with), move this assignment to the declaration and type: `errorText: string | null = null`.

Reply:
took it out